cypress/integration/teacher_demonstrations.js

```diff
@@ -1,84 +1,83 @@
-describe.skip('Teacher demonstrations test', () => {
```

Comment:
it looks like you reindented this whole file to 2-space indent from 4-space indent.

I'm not against the idea of changing the indentation standard for cypress tests from 4-spaces to 2-spaces if that's more standard in the community, but if we were to do that we should do it across all cypress tests at the same time, and do it in its own PR not mixed in with other changes. Please revert this conversion from this PR so that the PR only shows content changes to the test without changes to spacing mixed in

Reply:
My prettier extension tab indent was on 2. I changed it to 4. Its weird that it only did this to this specific file and not the others..